index.js

```diff
@@ -27,37 +29,58 @@ module.exports = function(robot) {
    */
   function help(res) {
     res.send(
-      '`deploy add _metadata_`: Add yourself to the deploy queue. Hubot give you a heads up when it\'s your turn. Anything after `add` will be included in messages about what you\'re deploying. Something like `hubot deploy add my_api`.\n' +
-      '`deploy done`: Say this when you\'re done and then Hubot will tell the next person. Or you could say `deploy complete`.\n' +
-      '`deploy remove _user_`: Removes a user completely from the queue. Use `remove me` to remove yourself. Also works with `deploy kick _user_`.\n' +
-      '`deploy next`: Sneak peek at the next person in line. Also works with `deploy who\'s next` and `deploy who\'s on first`.\n' +
-      '`deploy list`: Lists the queue.\n' +
-      '`deploy debug`: Kinda like `deploy list`.\n'
+      "`deploy add _metadata_`: Add yourself to the deploy queue. Hubot give you a heads up when it's your turn. Anything after `add` will be included in messages about what you're deploying. Something like `hubot deploy add my_api`.\n" +
+        "`deploy done`: Say this when you're done and then Hubot will tell the next person. Or you could say `deploy complete`.\n" +
+        "`deploy remove _user_`: Removes a user completely from the queue. Use `remove me` to remove yourself. Also works with `deploy kick _user_`.\n" +
+        "`deploy next`: Sneak peek at the next person in line. Also works with `deploy who's next` and `deploy who's on first`.\n" +
+        "`deploy list`: Lists the queue.\n" +
+        "`deploy debug`: Kinda like `deploy list`.\n"
```

Comment:
Prettier ran wild, I can revert this sort of thing if it seems bad.

Reply:
It would be good to split the formatting and actual changes

Reply:
Yeah I decided so too! I think I'm just going to leave those changes out.